samples/openapi/main.go

```diff
@@ -145,7 +145,7 @@ func openAPIGetStructAuditTrail(vcdClient *govcd.VCDClient) {
 	filterTime := time.Now().Add(-12 * time.Hour).Format(types.FiqlQueryTimestampFormat)
 	queryParams.Add("filter", "timestamp=gt="+filterTime)
 
-	err = vcdClient.Client.OpenApiGetAllItems("33.0", urlRef, queryParams, &response, nil)
+	err = vcdClient.Client.OpenApiGetAllItems("34.0", urlRef, queryParams, &response, nil)
```

Comment:
same here

Reply:
We can set this. It would probably complain about it now.